pyiron_contrib/atomistic/mlip/mlip.py

```diff
@@ -388,10 +397,10 @@ def load_default(self, file_content=None):
         if file_content is None:
             file_content = '''\
 $MLP_COMMAND_PARALLEL train --energy-weight=energy_auto --force-weight=force_auto --stress-weight=stress_auto --max-iter=iteration_auto start.mtp training.cfg > training.log
-$MLP_COMMAND_SERIAL calc-grade Trained.mtp_ training.cfg testing.cfg grades.cfg --mvs-filename=state.mvs > grading.log
-$MLP_COMMAND_SERIAL select-add Trained.mtp_ training.cfg testing.cfg diff.cfg > select.log
-cp training.cfg training_new.cfg 
-cat diff.cfg >> training_new.cfg
+# $MLP_COMMAND_SERIAL calc-grade Trained.mtp_ training.cfg testing.cfg grades.cfg --mvs-filename=state.mvs > grading.log
```

Comment:
Are these not available in the legacy version?

Reply:
No - but I made it optional in the latest commit. They have some calc-error function but the syntax is different, so for the scope of the workshop I would suggest to validate the fit manually, as we anyway can not compare the residual computed by different codes.